docs/networking.md

```diff
@@ -206,10 +206,22 @@ For help with Calico or to report any issues:
 
 #### Calico Backend
 
-In kops 1.12.0 and later Calico uses the k8s APIServer as its datastore. The current setup does not make use of [Typha](https://github.com/projectcalico/typha) - a component intended to lower the impact of Calico on the k8s APIServer which is recommended in [clusters over 50 nodes](https://docs.projectcalico.org/latest/getting-started/kubernetes/installation/calico#installing-with-the-kubernetes-api-datastoremore-than-50-nodes) and is strongly recommended in clusters of 100+ nodes.
+In kops 1.12.0 and later Calico uses the k8s APIServer as its datastore.
 
 In versions <1.12.0 of kops Calico uses etcd as a backend for storing information about workloads and policies. Calico does not interfere with normal etcd operations and does not require special handling when upgrading etcd.  For more information please visit the [etcd Docs](https://coreos.com/etcd/docs/latest/)
 
+#### Configuraing Calico to use Typha
+
+As of Kops 1.12 Calico uses the kube-apiserver as its datastore. The default setup does not make use of [Typha](https://github.com/projectcalico/typha) - a component intended to lower the impact of Calico on the k8s APIServer which is recommended in [clusters over 50 nodes](https://docs.projectcalico.org/latest/getting-started/kubernetes/installation/calico#installing-with-the-kubernetes-api-datastoremore-than-50-nodes) and is strongly recommended in clusters of 100+ nodes.
+It is possible to configure Calico to use Typha by editing a cluster and adding a
+`typhaReplicas` option to the Calico spec:
+
+```
+  networking:
+    calico:
+      typhaReplicas: 3
```

Comment:
Can an existing calico be upgraded to typha without incident? .. I'm guessing no and if so, perhaps just chunk in a helpful comment indicating.

Reply:
I'll need to double check if it's still the case, but it used to be the case that they could (~ version 3.2 from memory) as the new calico node pods come up and see the new value for `FELIX_TYPHAK8SSERVICENAME` they connect to Typha rather than directly to the APIServer.

I'll have a play around with that at some point over the next couple of days to confirm though.

Reply:
Agreed that this is an important thing for us to test and confirm. I feel like quite a few people will be excited for this and jump in when they see it in the release notes!